src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+class Node<T> {
+    private T data;
+    private Node<T> next;
+    private Node<T> prev;
+
+    Node(T data) {
+        this.data = data;
+        this.next = null;
+        this.prev = null;
+    }
+
+    public T getData() {

```

Comment:
[Don't use getters and setters in class Node](https://github.com/mate-academy/jv-linked-list/blob/master/checklist.md#dont-use-getters-and-setters-in-class-node)